packages/Autodiscovery/tests/Yaml/ExplicitToAutodiscoveryConverterTest.php

```diff
@@ -65,8 +62,8 @@ private function doTestFile(string $file, int $nestingLevel): void
      */
     private function splitFile(string $yamlContent): array
     {
-        if (Strings::match($yamlContent, self::SPLIT_PATTERN)) {
-            return Strings::split($yamlContent, self::SPLIT_PATTERN);
+        if (Strings::match($yamlContent, AbstractCheckerTestCase::SPLIT_LINE)) {
+            return Strings::split($yamlContent, AbstractCheckerTestCase::SPLIT_LINE);
```

Comment:
Sorry, my bad. I didn't notice it's a different package here, Autodiscovery.
In that case, it has to have own constant, because ECS will not be present

Reply:
Though keeping `-----` united split constant would be great. Each package with their own constant

Reply:
Done